### ISSUE TITLE
"key ports not found" error when restoring headless service without port

### ISSUE THREAD

**Issue by spingel** [heptio/velero#1154]
**Steps:**
1. Backup headless service without port definition
2. Restore headless service

**Error:**
```
./ark restore describe my-restore
...
  Namespaces:
    test:  error preparing /tmp/801038680/resources/services/namespaces/test/headless-service.json: rpc error: code = Unknown desc = key ports not found
```

**Service:**
```yaml
  kind: Service
  apiVersion: v1
  metadata:
    name: headless-service
    namespace: test
  spec:
    clusterIP: None
    selector:
      app: test
```

- Ark version (use `ark version`): 0.10.0
- Kubernetes version (use `kubectl version`): 1.12
- Kubernetes installer & version: 1.10.11

**Comment by ncdc** [comment]
@spingel thanks for the report. This will be fixed by #1146

**Comment by skriss** [comment]
#1146 has been merged so I'm closing this issue. This will be released in v0.12.
